docs/source/customizations/coprocessor.mdx

```diff
@@ -298,7 +299,115 @@ Properties of the JSON body are divided into two high-level categories:
 {
   // Control properties
   "version": 1,
-  "stage": "SupergraphRequest",
+  "stage": "SupergraphResponse",
```

Comment:
I can't tell from the diff if this is correct, but it seems odd that `SupergraphRequest` is now `SupergraphResponse`. Is this fixing an existing problem?

Reply:
the section title is about the `SupergraphResponse` stage, but it looks like the response is wrong, because we would not have a `control` field, right?

Reply:
there's actually a control field on the supergraph response from coprocessor, to set the status